### PR TITLE
Added `Choice` and `Strong`.

### PR DESCRIPTION
**Pull request by joneshf**

### CODE REVIEW

File: src/Data/Profunctor/Strong.purs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Data.Profunctor.Strong where
+
+  import Data.Profunctor (Profunctor)
+  import Data.Tuple (Tuple(..))
+
+  class (Profunctor p) <= Strong p where
+    first' :: forall a b c. p a b -> p (Tuple a c) (Tuple b c)
```
Comment (paf31) [inline src/Data/Profunctor/Strong.purs]: Just out of curiosity, why the ticks here?

Comment (joneshf) [inline src/Data/Profunctor/Strong.purs]: Avoid conflicts with https://github.com/purescript-contrib/purescript-arrows/blob/master/src/Control/Arrow.purs#L7

**Comment by joneshf** [comment]
bump

**Comment by paf31** [comment]
Looks fine to me. I'm a little sad about the ticks in the names though. We really need qualified names for type class members.

@garyb ?

**Comment by garyb** [comment]
I think there's something wrong with GitHub notifications for me, or I'm oblivious ;) didn't even notice this before...

Is it really a problem to call it `first`? I'd rather qualify on import personally. Isn't `Profunctor` more fundamental than `Arrow` too, so if either should be prime'd it should be `Arrow`.

**Comment by joneshf** [comment]
Guess it's time to read http://www-mmm.is.s.u-tokyo.ac.jp/~ichiro/papers/fromComptoComp.pdf and friends.

**Comment by garyb** [comment]
Can we just go ahead and change these to `first` and `second` or is that going to cause problems that can't be fixed by qualification?

**Comment by garyb** [comment]
Thanks! Sorry it took so long to get to...

**Comment by joneshf** [comment]
No worries :).

**Pull request merged**